content/en/docs/components/model-registry/installation.md

```diff
@@ -18,26 +18,35 @@ These are the minimal requirements to install Model Registry:
 
 ## Installing Model Registry
 
-You can skip this step if you have already installed Kubeflow >=1.9. Your Kubeflow
-deployment includes Model Registry ([see tracker issue](https://github.com/kubeflow/manifests/issues/2631)).
+Kubeflow Model registry may be installed as part of a Kubeflow Platform, or as a standalone component.
+The best option for you will depend on your specific requirements.
 
-To install Model Registry as part of Kubeflow, follow the
-[Kubeflow installation guide](/docs/started/installing-kubeflow/).
+### Installing on Kubeflow Platform
+
+Kubeflow Model Registry is available as an opt-in alpha component in Kubeflow Platform 1.9+, see [Installing Kubeflow](/docs/started/installing-kubeflow/) to learn more about deploying the Kubeflow Platform.
+
+If you have deployed the raw manifests, you may follow [these instructions](https://github.com/kubeflow/manifests/tree/master/apps/model-registry/upstream#readme) to deploy Model Registry; please raise any feedback on [`kubeflow/manifests#2631`](https://github.com/kubeflow/manifests/issues/2631).
+
+### Standalone installation
 
 If you want to install Model Registry separately from Kubeflow, or to get a later version
-of Model Registry, you can use one of the following Model Registry manifests.
-Remember to substitute the relevant release (e.g. `v0.1.2`), modify `ref=main` to `ref=v0.1.2`.
+of Model Registry, you can the Model Registry manifests directly from the [Model Registry repository](https://github.com/kubeflow/model-registry).
+
+The manifests by defaut deploy the Model Registry in the `kubeflow` namespace;
+you must ensure the `kubeflow` namespace is available (e.g.: `kubectl create namespace kubeflow`)
+or modify the [kustomization file](https://github.com/kubeflow/model-registry/blob/v0.2.0-alpha/manifests/kustomize/overlays/db/kustomization.yaml#L3) accordingly to your setup.
 
-The following steps show how to install Model Registry in the context of a default Kubeflow >=1.8 installation.
+Remember to substitute the relevant release version number, by modifying the actual `ref=` to your desired version number.
+For example, modify `ref=v0.2.0-alpha` to `ref=v0.3.2` if you want to deploy version 0.3.2 instead.
 
 ```shell
-kubectl apply -k "https://github.com/kubeflow/model-registry/manifests/kustomize/overlays/db?ref=main"
+kubectl apply -k "https://github.com/kubeflow/model-registry/manifests/kustomize/overlays/db?ref=v0.2.0-alpha"
 ```
 
-As the default Kubeflow installation provides an Istio mesh, apply the necessary manifests:
+You may also apply the Istio-related manifests if your cluster installation provides an Istio-mesh; these are also required when using Model Registry as part of Kubeflow.
 
 ```shell
-kubectl apply -k "https://github.com/kubeflow/model-registry/manifests/kustomize/options/istio?ref=main"
+kubectl apply -k "https://github.com/kubeflow/model-registry/manifests/kustomize/options/istio?ref=v0.2.0-alpha"
```

Comment:
Can users take the kustomize manifests from `main` branch to install the latest changes ?
Or you push images to the DockerHub only when you do release tag ?

Reply:
done with https://github.com/kubeflow/website/pull/3740/commits/b75772fd609ddf28679c1708ad519d2cd192dcab

Reply:
Resolving this thread based on feedback from : https://github.com/kubeflow/website/pull/3740#pullrequestreview-2093485477